tests/integration/targets/nxos_config/tests/cli/replace_block.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- ansible.builtin.debug: msg="START cli/replace_block.yaml on connection={{ ansible_connection }}"
+
+- name: "Populate nxos acls configuration with replace block and lines options"
+  cisco.nxos.nxos_config:
+    lines:
```

Comment:
Can we do a `lookup()` instead on the same template, and assert that the changes are the same in both cases?

Reply:
ACK